### DISCUSSION TITLE
lovelace-views - Incorrect type. Expected "object".

### DISCUSSION THREAD

**Discussion by slezak-petr** [hassio-addons/addon-vscode#518] [Q&A]
Hello
I have lovelace in splited yaml files

**ui-lovelace.yaml**
_views: !include_dir_merge_list lovelace/views/_

**lovelace/views/lovelace_file.yaml**

- title: My title of view
  icon: mdi:numeric
  cards:
    - type: entities
      title: Example
      entities:
        - light.kitchen
        - switch.ac
        - entity: light.living_room
          name: LR Lights

By Visual studio I get error:
**Incorrect type. Expected "object".
yaml-schema: http://schemas.home-assistant.io/lovelace-views**

**But lovelace WORKS**

I was tried changed:

**ui-lovelace.yaml**
views: !include_dir_merge__**named** lovelace/views/

and in **lovelace/views/lovelace_file.yaml** change to:
(without a dash at the beginning of the code)

title: My title of view
icon: mdi:numeric
cards:
  - type: entities
    title: Example
    entities:
    ...

**Than for Visual studio Code its OK, 
but lovelace does not work**

**Comment by sjoshua270** [comment]
Same exact issue here.
It's nearly unusable, as all my yaml view files are covered in error underlines.
